src/export/pdf/index.ts

```diff
@@ -32,9 +33,11 @@ export async function articleToPdf(
   });
 
   const buildPath = path.join(outputPath, '_build');
-  const CMD = `cd ${buildPath};latexmk -f -xelatex -synctex=1 -interaction=nonstopmode -file-line-error -latexoption="-shell-escape" ${tex_filename}`;
+  const CMD = `cd ${buildPath};latexmk -f -xelatex -synctex=1 -interaction=batchmode -file-line-error -latexoption="-shell-escape" ${tex_filename} &> ${tex_log_filename}`;
```

Comment:
@stevejpurves this is the main change. Putting this to a file.

The main.log is still made, and I don't think the console output is different, but it is there if we want it.

Reply:
Yeah, does everrything nonstop does but doesn't print as many logs.